### ISSUE TITLE
Testnet-3 occured panic error, when manully node

### ISSUE THREAD

**Issue by retz80** [KiraCore/testnet#45]
I tried to synchronize nodes and an error occurred.

```
5:29AM INF Dialing peer address={"id":"0d29dc6c0da53d4d5d450bd28b5e10008ada1049","ip":"121.78.241.72","port":26656} module=p2p
5:29AM INF Stopping gossipDataRoutine for peer module=consensus peer={"Data":{},"Logger":{}}
5:29AM INF Stopping gossipVotesRoutine for peer module=consensus peer={"Data":{},"Logger":{}}
panic: Failed to process committed block (1:F0977946C6B64030ECA9FDDD364691EF9B5B5F9EF377E74B2F73678FBF9BE00A): block time 2021-05-02 17:55:17.335672856 +0000 UTC is not equal to genesis time 2021-05-02 15:48:35.606923479 +0000 UTC

goroutine 98 [running]:
github.com/tendermint/tendermint/blockchain/v0.(*BlockchainReactor).poolRoutine(0xc000d1d500, 0x0)
        /home/ret/go/pkg/mod/github.com/tendermint/tendermint@v0.34.9/blockchain/v0/reactor.go:401 +0x1605
created by github.com/tendermint/tendermint/blockchain/v0.(*BlockchainReactor).OnStart
        /home/ret/go/pkg/mod/github.com/tendermint/tendermint@v0.34.9/blockchain/v0/reactor.go:110 +0x8c
```

I checked genesis.json checksum, again.

```
$ sha256sum genesis.json
e800493110fb1e0b8cc3360643387eb05d0d3b08a22212dbacaccf9c1308ea8e  genesis.json
```

It was installed from the source and runned from no block data.
as I know, if genesis file is different, it should not be connected, not occured panic error.

**Comment by westaking** [comment]
The same thing happens to me.

If the genesis are different then you shouldn't try to synchronize.
Also, do not panic and skip the node along with the error log.

If it download and install snapshot, and synchronize it, it will be synchronized without any problems.

Apart from the above problem, there is only one peer connected among the numerous nodes in peers.txt & snaps.txt.

**Comment by asmodat** [comment]
Resolved with v0.3.2.3 - `testnet-3`
